packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/RuntimeUpgrade.tsx

```diff
@@ -76,24 +97,30 @@ export const RuntimeUpgrade = ({ setRuntime }: RuntimeUpgradeProps) => {
             </RowGapBlock>
             <TextSmall lighter>Maximum upload file size is 3 MB</TextSmall>
           </RowGapBlock>
-          {!!acceptedFiles.length && (
-            <RowGapBlock gap={8}>
-              {acceptedFiles.map((file) => (
-                <ReceivedFile key={file.name} valid={true}>
-                  <AcceptedFileText>
-                    <b>{file.name}</b> ({file.size} B) was uploaded successfully!
-                  </AcceptedFileText>
-                </ReceivedFile>
-              ))}
-              {fileRejections.map(({ file, errors }) => (
-                <ReceivedFile key={file.name} valid={false}>
-                  <AcceptedFileText>
-                    <b>{file.name}</b> ({file.size} B) was not loaded because of "{errors.map((e) => e.message)}"
-                  </AcceptedFileText>
-                </ReceivedFile>
-              ))}
-            </RowGapBlock>
-          )}
+          <RowGapBlock gap={8}>
+            {acceptedFiles.map((file) => (
+              <ReceivedFile key={file.name} valid={true}>
+                <AcceptedFileText>
+                  <b>{file.name}</b> ({file.size} B) was loaded successfully!
+                </AcceptedFileText>
+              </ReceivedFile>
+            ))}
+            {fileRejections.map(({ file, errors }) => (
+              <ReceivedFile key={file.name} valid={false}>
+                <AcceptedFileText>
+                  <b>{file.name}</b> ({file.size} B) was not loaded because of "{errors.map((e) => e.message)}"
+                </AcceptedFileText>
+              </ReceivedFile>
+            ))}
+            {fileValidation && !fileValidation.isValid && (
+              <ReceivedFile key={fileValidation.file.name} valid={false}>
+                <AcceptedFileText>
+                  <b>{fileValidation.file.name}</b> ({fileValidation.file.size} B) was not loaded because it could be
+                  malicious
+                </AcceptedFileText>
```

Comment:
```suggestion
                <AcceptedFileText>
                  <b>{fileValidation.file.name}</b> ({fileValidation.file.size} B) was not loaded because it is not
                  valid WASM file.
                </AcceptedFileText>
```

Reply:
Actually, this is to be removed when using a validator.